images/docs-builder/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+if [ $# -lt 1 ]; then
+	echo "You need to provide an argument [build_docs, build_docs_pdf, check_links, docs_serve]"
+fi
+
+case $1 in
+	build_docs)
+		echo "Generating html from docs source"
+		shift 
+		asciidoctor $@
+		;;
+	build_docs_pdf)
+		echo "Generating pdf from docs source"
+		shift 
+		asciidoctor-pdf $@
+		;;
+	check_links)
+		echo "Checking if all links are alive in docs source"
+		find . -name \*.adoc | xargs -n1 asciidoc-link-check -q
+		;;
```

Comment:
Minor potential refinement, this 'check_links' code could be added at the same time as you add support for it in the Makefile/CI (just a comment, no changes required). Seeing that ' Add make target to check links in the docs ' modifies this part of the script, better to have all the changes in a single place

Reply:
Fwiw, all other targes have 'docs' in the name, either I'd expect a 'docs' mention here, or I'd remove it from the other targets as this is a 'doc-builder' image already.

Reply:
Changed to `docs_check_links`